chain/network/src/peer_manager/tests/routing.rs

```diff
@@ -28,9 +28,19 @@ use std::collections::HashSet;
 use std::net::Ipv4Addr;
 use std::sync::Arc;
 
+fn abort_on_panic() {
```

Comment:
Hm, I think this would be better off as a separate PR. There some issues with this hook, which I don't believe need to block the port binding changes in this PR.

Reply:
moved to https://github.com/near/nearcore/pull/8439